ironfish/src/rpc/routes/chain/getTransactionStream.ts

```diff
@@ -147,20 +211,30 @@ router.register<typeof GetTransactionStreamRequestSchema, GetTransactionStreamRe
       })
     }
 
+    const validatedAssetValue = async (assetIdentifier: Buffer): Promise<AssetsValue> => {
+      const assetValue = await node.chain.assets.get(assetIdentifier)
+      if (!assetValue) {
+        throw new ValidationError(
+          `Asset detected in chain is not in assetDB. Asset Identifier: ${assetIdentifier.toString()}`,
+        )
+      }
```

Comment:
This will crash incorrectly in the case that an asset has removed, we need to remove this error and handle this properly to avoid errors / crashes. Let's think about this case carefully